### PR TITLE
Release branch fix snapshot

### PR DESCRIPTION
**Pull request by lxfind**
## Description 

Describe the changes or additions included in this PR.

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and do not break anything, you can skip the following section. Otherwise, please briefly describe what has changed under the Release Notes section.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: crates/sui-protocol-config/src/snapshots/sui_protocol_config__test__Mainnet_version_37.snap [hunk 1]
```diff
@@ -207,7 +206,7 @@ check_zklogin_id_cost_base: 200
 check_zklogin_issuer_cost_base: 200
 scoring_decision_mad_divisor: 2.3
 scoring_decision_cutoff_value: 2.5
-execution_version: 3
+execution_version: 2
```
Comment (mystenmark) [inline crates/sui-protocol-config/src/snapshots/sui_protocol_config__test__Mainnet_version_37.snap:209]: how did the wrong execution version end up in the release branch?

**Review by mystenmark — APPROVED** [review]

**Comment by ebmifa** [comment]
Ninja merged into the `releases/sui-v1.20.0-release`branch as `d951895b90014b58cfd7734e3a5a25d7a37ee9b7`

File: crates/sui-protocol-config/src/snapshots/sui_protocol_config__test__Mainnet_version_37.snap (hunk 1 above)
Comment (dariorussi) [reply crates/sui-protocol-config/src/snapshots/sui_protocol_config__test__Mainnet_version_37.snap:209]: that is so bad... sorry...
I forced execution version to a previous protocol config just to try the fix and I do not understand how it got back into the PR. 
I thought I did that on my machine only, but obviously....

**Comment by dariorussi** [comment]
are those just snapshot fixes? 
so the snapshot got messed up?

**Pull request closed without merging**